sdk/lusid/models/weighted_instruments.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, instruments=None):  # noqa: E501
         WeightedInstruments - a model defined in OpenAPI
 
         :param instruments:  The instruments that are held in the set. (required)
-        :type instruments: list[lusid.WeightedInstrument]
+        :type instruments: 
```

Comment:
We need to make sure these types haven't dissapeared?

Reply:
I'm not sure how close I've got with the types. It doesn't seem like there are as many Mustache variables with the python-legacy generator to properly construct the types.